.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,38 @@
+landing_getting_started_1: |-
+    # Configure your entities by adding them in config/packages/meili_search.yaml
+    meili_search:
+        url: 'http://127.0.0.1:7700'
+        api_key: 'masterKey'
+        indices:
+            - name: movies
+              class: App\Entity\Movie
+
+    <?php
+    // src/Controller/MoviesController.php
+    namespace App\Controller;
+
+    use App\Entity\Movie;
+    use Doctrine\Persistence\ManagerRegistry;
+    use Symfony\Component\HttpFoundation\Response;
+
+    class MoviesController extends AbstractController
+    {
+        #[Route('/add-movies', name: 'addMovies')]
+        public function addMovies(ManagerRegistry $doctrine): Response
+        {
+            $entityManager = $doctrine->getManager();
```

Comment:
I would like to just call the persistence and say this will be automatically indexed, but I don't know if this would be clear for symfony users. WDYT @curquiza & @gmourier ?

Reply:
I'm not sure to understand, for me you already said it will be automatically updated some line below right?
```
// Inserting data in your DB table will automatically update your Meilisearch index
```

Reply:
Yeah, but do you think this comment is necessary?

Reply:
I don't see any problem to have it on a sample code to remind this information. Will the content of this comment be mentioned before or after the sample code?

If not, it seems useful to me because there is a bit of magic with the ORM hooks here, it also helps someone who doesn't necessarily read the surrounding text and copy and paste the sample code directly.

Reply:
It's ok from my POV to let it. We added it to Rails as well, and I agree with guillaume: it's a little bit magic and not obvious, from my POV